src/defaultsettings.cpp

```diff
@@ -206,6 +206,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("invert_mouse", "false");
 	settings->setDefault("mouse_sensitivity", "0.2");
 	settings->setDefault("repeat_rightclick_time", "0.25");
+	settings->setDefault("safe_dig_and_place", "false");
```

Comment:
Interesting point, let's discuss this in the general comments below (change requests have a tendency to turn invisible when rebasing and such).